tests/conftest.py

```diff
@@ -431,21 +447,21 @@ async def clean_db(postgresql_pool, create_db, postgres_db):
     """
     yield
     # By using the connection pool, we can make sure that the connection we use is alive
-    async with postgresql_pool.acquire() as postgresql_client:
-        tables_in_db = await postgresql_client.fetch(
-            "SELECT table_name FROM information_schema.tables WHERE table_schema='public'"
-        )
-        tables_in_db = [x["table_name"] for x in tables_in_db]
-        tables_to_preserve = TABLES_TO_KEEP
-        tables_to_preserve.append(SCHEMA_VERSION_TABLE)
-        tables_to_truncate = [x for x in tables_in_db if x in tables_to_preserve and x != SCHEMA_VERSION_TABLE]
-        tables_to_drop = [x for x in tables_in_db if x not in tables_to_preserve]
-        if tables_to_drop:
-            drop_query = "DROP TABLE %s CASCADE" % ", ".join(tables_to_drop)
-            await postgresql_client.execute(drop_query)
-        if tables_to_truncate:
-            truncate_query = "TRUNCATE %s CASCADE" % ", ".join(tables_to_truncate)
-            await postgresql_client.execute(truncate_query)
+
+    tables_in_db = await postgresql_client.fetch("SELECT table_name FROM information_schema.tables WHERE table_schema='public'")
+    tables_in_db = [x["table_name"] for x in tables_in_db]
+    tables_to_preserve = TABLES_TO_KEEP
+    tables_to_preserve.append(SCHEMA_VERSION_TABLE)
+    tables_to_truncate = [x for x in tables_in_db if x in tables_to_preserve and x != SCHEMA_VERSION_TABLE]
+    tables_to_drop = [x for x in tables_in_db if x not in tables_to_preserve]
+    if tables_to_drop:
+        drop_query = "DROP TABLE %s CASCADE" % ", ".join(tables_to_drop)
+        await postgresql_client.execute(drop_query)
+    if tables_to_truncate:
+        truncate_query = "TRUNCATE %s CASCADE" % ", ".join(tables_to_truncate)
+        await postgresql_client.execute(truncate_query)
+
+    await postgresql_client.close()
```

Comment:
bit odd to close a client we didn't create?

Reply:
oops, not intended, nice catch.